Dockerfile

```diff
@@ -1,33 +1,38 @@
-FROM alpine:3.4
-MAINTAINER Thomas Boerger <thomas@webhippie.de>
+#Build stage
+# TODO ARG GO_VERSION=edge
+#FROM TODO multiarch/golang:amd64-${GO_VERSION} AS build-env
+#FROM webhippie/golang:${GO_VERSION} AS build-env
+FROM sapk/alpine-multi AS build-env
+
+ARG TAGS
+ENV TAGS "bindata $TAGS"
+ENV GOPATH /go/
+ENV PATH "${PATH}:${GOPATH}/bin"
+
+RUN apk -U --no-cache add go git build-base \
+ && go get -u github.com/jteeuwen/go-bindata/...
+
+ADD . ${GOPATH}/src/code.gitea.io/gitea
+WORKDIR ${GOPATH}/src/code.gitea.io/gitea
+
+RUN pwd && ls -lah && make clean generate build
+
+
+#Run stage
+#ARG ALPINE_VERSION=latest
+FROM sapk/alpine-multi
+LABEL maintainer "Thomas Boerger <thomas@webhippie.de>"
+
+ARG UID=1000
+ARG GID=1000
 
 EXPOSE 22 3000
 
-RUN apk update && \
-  apk add \
-    su-exec \
-    ca-certificates \
-    sqlite \
-    bash \
-    git \
-    linux-pam \
-    s6 \
-    curl \
-    openssh \
-    tzdata && \
-  rm -rf \
-    /var/cache/apk/* && \
```

Comment:
https://github.com/gliderlabs/docker-alpine/blob/master/docs/usage.md#disabling-cache It seem, it doesn't even need -U also.